### PR TITLE
KISS-kde wiki page added

### PR DESCRIPTION
**Pull request by dilyn-corner**
This is a draft of a wiki page for KISS-kde. 

The releases still need to be uploaded to the repository so that the links are live (they're currently building), but until then I would like to get feedback so I can make changes while I wait on these horrendous build times :)

### CODE REVIEW

**Comment by sdsddsd1** [comment]
I think this is allready a good read and well structured. However I would like to see the "from scratch setup" first in the list, as this should be the preferred one. KISS is a source based distro and this should get honored first. Altough this is a huge project with large times in building stuff, imho it is not the best practice in terms of security to share binaries.

File: desktops/kde.txt [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+    +--------------------------------------------------------------------------+
+
+    Take care of any circular dependencies:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b elogind                                                       |
+    |   $ kiss i elogind                                                       |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
```
Comment (sdsddsd1) [inline desktops/kde.txt:257]: I would like to see where between which packages this cirular dependency is.

File: desktops/kde.txt [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ ln -s /etc/sv/dbus  /var/service                                     |
+    |   $ ln -s /etc/sv/udevd /var/service                                     |
+    |                                                                          |
+    |   $ sv up dbus                                                           |
+    |   $ sv up udevd                                                          |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
```
Comment (sdsddsd1) [inline desktops/kde.txt:269]: Enabeling theses services is also mentioned in the paragraph `login manager`.  
Maybe only one paragraph can be about which services are required for KDE and how to set them up.

Comment (dilyn-corner) [reply desktops/kde.txt:269]: These specific services are only mentioned once; the ones listed later are for `sddm` to run, and neither `dbus` nor `eudev` are required for that. 
A services section seems overkill; the reasons for running these services is perhaps less self-evident than I think they are, but no services are actually required to run Plasma.

File: desktops/kde.txt (hunk 1 above)
Comment (dilyn-corner) [reply desktops/kde.txt:257]: I believe I saw a circularity between `elogind` and `pam`; I will double check that.

**Comment by dilyn-corner** [comment]
I agree that bianry-sharing should be discouraged; I'm unsure if there is a way to improve the security of that? 

I will privilege the source section over the tarball.

File: desktops/kde.txt (hunk 2 above)
Comment (sdsddsd1) [reply desktops/kde.txt:269]: My bad. I mixed that up with polkitd, elogind,... .  :see_no_evil: 
That no services are required to run `plasma` is a valueable information!

**Comment by sdsddsd1** [comment]
` I'm unsure if there is a way to improve the security of that?`
I think we cant. This always depends on trust. We already trust one person for a clean chroot and firefox-bin.

File: desktops/kde.txt (hunk 2 above)
Comment (dilyn-corner) [reply desktops/kde.txt:186]: I've included a small note about service requirements!

**Comment by dilyn-corner** [comment]
Nobody should trust me. I'm a very shady character! 
I could do something like Dylan does and provide signatures and my public key but that really just moves the trust problem down a few rows of abstraction. Do you trust me, saying that I'm trustworthy, or do you trust me AND these three other people who signed my key saying I am AND that nobody has yet to say I'm untrustworthy... 
Providing a public key at least shows good faith. But ultimately, it does just come down to not thinking I'm a bad actor. 
For those amongst us who aren't so trusting, they can always just wait fifteen hours for `webengine` (:

File: desktops/kde.txt (hunk 1 above)
Comment (dilyn-corner) [reply desktops/kde.txt:174]: Upon further examination, it does not seem that this is the case. I will drop this. Thank you!

**Comment by dilyn-corner** [comment]
I was going to say that this was ready to be merged, but there is going to be a shift towards `wayland` as the default with `plasma 5.20`. I'm going to be testing this, and this article will be expanding to discuss it.

**Comment by dylanaraps** [comment]
Will go over all of the pending PRs here soon. Sorry for the delay.

File: desktops/kde.txt [hunk 3]
```diff
@@ -0,0 +1,434 @@
+KDE
+________________________________________________________________________________
+
+KDE [0] is desktop environment which consists in the Plasma Desktop, the KDE 
+Frameworks, and a collection of free and open source, cross-plastform  programs.
```
Comment (dylanaraps) [inline desktops/kde.txt:5]: `cross-plastform` -> `cross-platform`

Two spaces between above and "programs".

File: desktops/kde.txt [hunk 4]
```diff
@@ -0,0 +1,434 @@
+KDE
+________________________________________________________________________________
+
+KDE [0] is desktop environment which consists in the Plasma Desktop, the KDE
```
Comment (dylanaraps) [inline desktops/kde.txt:4]: `consists in` -> `consists of` ?

File: desktops/kde.txt [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+    that time by ~800 to get a rough estimate of the time in minutes.
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b eudev dbus coreutils gnugrep                                  |
+    |   $ kiss i eudev dbus coreutils gnugrep                                  |
+    |   $ kiss a coreutils /usr/bin/realpath                                   |
+    |   $ kiss a coreutils /usr/bin/mktemp                                     |
+    |   $ kiss a coreutils /usr/bin/ln                                         |
+    |   $ kiss a gnugrep   /usr/bin/grep                                       |
```
Comment (dylanaraps) [inline desktops/kde.txt:112]: Maybe explain why the `kiss a` commands are required to readers? 

(Also, can these gnu-isms be fixed? Am happy to help.)

File: desktops/kde.txt [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+    |   $ kiss b sddm && kiss i sddm                                           |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    Then, enable the relevant services. polkit and elogind should already be
+    installed if you followed the prior instructions. If not, install them in
+    the usual way.
+    
+    NOTE: by default, it may not be possible for root to login via sddm. Best
+    practices would dictate that you login as a nonroot user.
```
Comment (dylanaraps) [inline desktops/kde.txt:312]: `nonroot` -> `non-root` (or maybe "normal"?)

File: desktops/kde.txt [hunk 7]
```diff
@@ -0,0 +1,434 @@
...
+    However, any features that appear in kwinft can be expected to eventually
+    show up in kwin. So if you prefer a more bleeding edge, potentially better
+    window manager for KDE, kwinft is an option. You can either swap to kwinft
+    from an existing KDE install or start immediately with using it. All you are
+    required to do is uncomment the kwinft lines in the dependencies list of
+    plasma-{desktop,workspace}, and comment out kwin in the same packages. Then,
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ pkill x                                                              |
```
Comment (dylanaraps) [inline desktops/kde.txt:353]: Isn't this usually `X` and not `x`?

**Review by dylanaraps — COMMENTED** [review]
This is awesome. Apologies for the delay in reviewing this.

File: desktops/kde.txt (hunk 6 above)
Comment (dilyn-corner) [reply desktops/kde.txt:312]: My background is showing. 
Does 'unprivileged' sound better?

File: desktops/kde.txt (hunk 5 above)
Comment (dilyn-corner) [reply desktops/kde.txt:112]: I can add that explanation.
If I am remembering right, `mktemp`, `grep`, and `ln` should be droppable. I just haven't worked on it (yet, was hoping to tackle it this week).

File: desktops/kde.txt (hunk 4 above)
Comment (dilyn-corner) [reply desktops/kde.txt:4]: Yet Another Typo!

File: desktops/kde.txt (hunk 3 above)
Comment (dilyn-corner) [reply desktops/kde.txt:5]: Fixed!

File: desktops/kde.txt (hunk 5 above)
Comment (dylanaraps) [reply desktops/kde.txt:112]: Nice. Fine to keep it as-is for now (can always update it later). Feel free to add a little explanation for `kiss a` if you deem it necessary.

File: desktops/kde.txt (hunk 6 above)
Comment (dylanaraps) [reply desktops/kde.txt:312]: Yes!

File: desktops/kde.txt (hunk 7 above)
Comment (dilyn-corner) [reply desktops/kde.txt]: I do believe it is. Fixed!

File: desktops/kde.txt (hunk 5 above)
Comment (dilyn-corner) [reply desktops/kde.txt:116]: I added a small reference!

**Comment by dylanaraps** [comment]
LGTM :+1:

**Comment by dylanaraps** [comment]
Thanks for doing this.

**Pull request merged**